src/app/client/src/app/modules/workspace/components/published/published.component.spec.ts

```diff
@@ -108,7 +111,7 @@ describe('PublishedComponent', () => {
   // if  search api's throw's error
   it('should throw error', inject([SearchService], (searchService) => {
     spyOn(searchService, 'compositeSearch').and.callFake(() => observableThrowError({}));
-    component.fetchPublishedContent(9, 1);
+    component.fetchPublishedContent(9, 1, { queryParams: { subject: ['english', 'odia'] }});
```

Comment:
The assertion is missing to check where the call to a method which fetches the content having these params or not